dask_conversions/done/Dask_Land_Change_geomedian.ipynb

```diff
@@ -4,7 +4,39 @@
    "cell_type": "code",
```

Comment:
<p>This will need to be uncommented - to enable running in fresh kernel.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SatelliteApplicationsCatapult/odc-hub/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Good catch</p>